buidl/cecc.py

```diff
@@ -449,3 +462,19 @@ def parse(cls, wif):
         else:
             raise ValueError("Invalid WIF")
         return cls(secret, network=network, compressed=compressed)
+
+
+def tagged_hash(tag, msg):
```

Comment:
Nice! Curious if this has a meaningful performance impact on the existing (non musig2) stuff.

Reply:
I haven't checked, but thought it would be better to be consistent. I know it'll make an impact once you get into 10k hashes or so.